ipn/ipnlocal/local.go

```diff
@@ -988,7 +988,11 @@ func (b *LocalBackend) WhoIs(ipp netip.AddrPort) (n tailcfg.NodeView, u tailcfg.
 	if !ok {
 		var ip netip.Addr
 		if ipp.Port() != 0 {
-			ip, ok = b.sys.ProxyMapper().WhoIsIPPort(ipp)
+			// TODO(andrew-d): is this right?
+			ip, ok = b.sys.ProxyMapper().WhoIsIPPort("tcp", ipp)
+			if !ok {
+				ip, ok = b.sys.ProxyMapper().WhoIsIPPort("udp", ipp)
+			}
```

Comment:
I don't love this; should we update the `WhoIs` call to require users to pass a protocol? That feels like it'll break things for ~all users... but we could have users pass a protocol but if there's none specified, default to "any protocol" like this? 😵‍💫

Reply:
Or default to TCP if they don't specify.

Reply:
@bradfitz just pushed a commit that takes an argument, but defaults to TCP; PTAL?